### PR TITLE
Moved installation notes to doc

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR moves the actual content to `Resources/doc/installation.md` and places general informations in the `README.md` file. This restructure allows us to extend the doc in the future without spaming in the `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,49 @@
 # SuluArticleBundle
 
+[![Build Status](https://travis-ci.org/sulu/SuluArticleBundle.svg?branch=master)](https://travis-ci.org/sulu/SuluArticleBundle)
+[![StyleCI](https://styleci.io/repos/61883398/shield?branch=develop)](https://styleci.io/repos/61883398)
+
+The SuluArticleBundle adds support for managing Articles in Sulu. Articles can be used in a lot of different ways to
```
Comment (danrot) [inline README.md]: `managing articles in Sulu`

File: README.md [hunk 2]
```diff
@@ -1,121 +1,49 @@
...
+manage unstructured data with an own URL in a admin-list. Most of the features which can be used in pages, can also be
```
Comment (danrot) [inline README.md]: `in an admin-list`

Comment (danrot) [inline README.md]: Most of the features, which

File: README.md [hunk 3]
```diff
@@ -1,121 +1,49 @@
...
+
+Additional features included:
+
+* Build in view-layer with elasticsearch
+* Segmentation of article-templates (called article-types)
+* Define URL schemas per type
+
+## Status
+
+This repository will become version 1.0 of SuluArticleBundle. It is under **heavy development** and currently it's APIs
```
Comment (danrot) [inline README.md]: `currently its APIs`

File: README.md [hunk 4]
```diff
@@ -1,121 +1,49 @@
...
+and code are not yet stable (pre 1.0).
```
Comment (danrot) [inline README.md]: `not stable yet`

File: README.md [hunk 5]
```diff
@@ -1,121 +1,49 @@
...
+* Composer
+* PHP `^5.5 || ^7.0`
+* Sulu `^1.4`
+* Elasticsearch `^2.2`
+
+For detailed requirements see [composer.json](https://github.com/sulu/SuluArticleBundle/blob/master/composer.json).
+
+## Documentation
+
+The the documentation is stored in the
```
Comment (danrot) [inline README.md]: `The documentation`

File: README.md [hunk 6]
```diff
@@ -1,121 +1,49 @@
...
-```bash
-bin/console sulu:document:init
-```
-
-Create elasticsearch index:
-
-```bash
-bin/console ongr:es:index:create
-```
+All the installation instructions are located in the documentation.
```
Comment (danrot) [inline README.md]: Would a link make sense?

Comment (wachterjohannes) [inline README.md]: yes (:

**Pull request merged**